stories/Templates/AuthorPage/AuthorPage.jsx

```diff
@@ -1,35 +1,47 @@
-import React, { useEffect } from 'react';
-import logowhite from '../../assets/images/undp-logo-white.svg';
-import logo from '../../assets/images/undp-logo-blue.svg';
-import user from '../../assets/images/user.svg';
-import viewport from '../../assets/js/viewport';
-import { Breadcrumbcomponent } from '../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs';
-import GlobalHeader from '../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader';
-import { AuthorSummary } from '../../Organism/Blocks/Authorcard/AuthorSummary';
-import { ContentCard } from '../../Organism/Blocks/ContentCard/ContentCard';
-import { Footer } from '../../Organism/Footer/Footer';
-import './AuthorPage.scss';
+import React, { useEffect } from "react";
+import logowhite from "../../assets/images/undp-logo-white.svg";
+import logo from "../../assets/images/undp-logo-blue.svg";
+import user from "../../assets/images/user.svg";
+import viewport from "../../assets/js/viewport";
+import { Breadcrumbcomponent } from "../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import GlobalHeader from "../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader";
+import { AuthorSummary } from "../../Organism/Blocks/Authorcard/AuthorSummary";
+import { ContentCard } from "../../Organism/Blocks/ContentCard/ContentCard";
+import { Footer } from "../../Organism/Footer/Footer";
+import "./AuthorPage.scss";
 
 const AuthorPage = ({
-  breadcrumbData, authorSummaryData, contentCardData, footerData,  languageswitcherData,
+  breadcrumbData,
+  authorSummaryData,
+  contentCardData,
+  footerData,
+
+  languageswitcherData,
+  navigationData,
   rightNavigationData,
   leftNavigationData,
-  navigationData,
   locale,
+  getMegaMenuData,
```

Comment:
@Ashish-srij I do not see `getMegaMenuData` being used anywhere. Can you remove it if it is not required.

Reply:
done